src/formatter/configuration/configuration.ts

```diff
@@ -53,6 +51,10 @@ class Configuration {
 
   delegates: Partial<Record<ContentType, Delegate>>;
 
+  get separator(): string {
```

Comment:
I would prefer `metadataSeparator` as this is used to join multiple metadata values.